.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,30 @@
+# CI stages to execute against Pull Requests
+name: Pylero - CI
+
+on:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+
+jobs:
+  codechecks:
+    name: Code Quality
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.8]
+    steps:
+      - name: Checkout Pylero
+        uses: actions/checkout@v2
+
+      - name: Set Up Python-${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependencies
+        run: |
+          pip install -U pip
+          pip install -U -r requirements.txt
+          pip install .
+      - name: Pre Commit Checks
```

Comment:
Unfortunately, we do have a lot of users installing pylero with py2.7. Can you please include that in the matrix but exclude this step?

Rationale being, we want to check installation is going well even with py2.7 and at the same time don't want all precommit checks with it, sounds good?